plugins/package-managers/conan/src/main/kotlin/Conan.kt

```diff
@@ -180,7 +180,7 @@ class Conan(
                 dependencies = parseDependencyTree(pkgInfos, projectInfo.buildRequires, workingDir)
             )
 
-            val projectPackage = parseProjectPackage(pkgInfos, definitionFile, workingDir)
+            val projectPackage = generateProjectPackage(projectInfo, definitionFile, workingDir)
```

Comment:
Maybe also worth noting in the commit message: The caller's `projectInfo` equals the original function's `projectPackage`, so the inlining also avoids calculating the value again.